### ISSUE TITLE
Consorsbank SecurePlus App quits service since Android 13 Upgrade

### ISSUE THREAD

**Issue by manwithoutname** [PrivSec-dev/banking-apps-compat-report#168]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### App name

Consorsbank Secureplus

### Link to app

https://play.google.com/store/search?q=consorsbank+secureplus+app&c=apps&hl=de

### App version

1.6.0

### Country of the app

Germany

### Build Number

TP1A.221005.002.2022101800

### Device

- [ ] Pixel 7 Pro
- [ ] Pixel 7
- [ ] Pixel 6a
- [ ] Pixel 6 Pro
- [X] Pixel 6
- [ ] Pixel 5a
- [ ] Pixel 5
- [ ] Pixel 4a (5G)
- [ ] Pixel 4a
- [ ] Pixel 4 XL
- [ ] Pixel 4
- [ ] Pixel 3a XL (obsolete)
- [ ] Pixel 3a (obsolete)
- [ ] Pixel 3 XL (obsolete)
- [ ] Pixel 3 (obsolete)

### Google Play installed

- [X] Installed
- [ ] Not installed

### Google Play services Network permission revoked?

- [ ] Revoked
- [X] Not revoked
- [ ] I did not have Google Play services installed

### SafetyNet Enforcement

- [ ] Enforced
- [ ] Not enforced
- [X] Unsure

### Native code debugging

- [ ] Enabled
- [X] Disabled

### Exploit protection compatibility mode

- [ ] Enabled
- [X] Disabled

### Stock OS compatibility

- [ ] Works
- [ ] Does not work
- [X] Not tested

### Profile app tested in

- [X] Main user profile
- [X] Secondary user profile
- [X] I used a work profile device manager app (Please mention more details about it down below like the app)

### Description of the app's functionality

The application receives messages confirming the release of bank transactions.

### Are there any extra notes you think users should know about?

The App just opens and closes when be activated!

**Comment by akc3n** [comment]
@manwithoutname If you have time, may you please try testing your banking app version again following these steps to possible solutions:

~~https://akc3n.page/banking-issues~~ https://github.com/akc3n/akc3n.page/blob/main/content/posts/banking-app-issues.md

**Comment by manwithoutname** [comment]
@akc3n Thank you very much for your comment. I would like to do all this very much. However, since the last grapheneos update which I installed yesterday, the problem seems to have resolved itself. In that respect, I probably can't help anymore.

My installation is as follows:
1. user - default user / private section: no google services installed.
1st user - business user (via Shelter): google services installed. This user is also running banking apps, installed from Aurora Store.
2nd user (I call him "googleuser"; google services installed, banking apps installed.

Until upgrading to Android 13, all banking apps ran fine, both in businessuser and googleuser. With A13, some suddenly stopped running. consorsbank and renaultbank failed to work.

Yesterday I installed the graphenos update TP1A.221005.002.2022102300. I just reinstalled the bankingapps in both googleuser and businessuser. Both apps are now running fine in both users.
